lib/pod/command/package.rb

```diff
@@ -67,10 +67,13 @@ def build_in_sandbox(platform)
 
         sandbox = install_pod(platform.name)
 
-        perform_build(platform, sandbox)
```

Comment:
The `ensure` block will only be ran in case of exceptions, though, not when we exit the process. Might be better to throw an `InformationalError` from `xcodebuild` - then we would hit the `ensure` block.